From c87949e23e48a63a387485b2246896c8223c49ca Mon Sep 17 00:00:00 2001
From: Lukas Obermann <obermann.lukas@gmail.com>
Date: Mon, 4 Sep 2017 21:44:30 +0200
Subject: [PATCH] [imdb] partial revert of crashfix due to a introduced bug

one part of the fix for the crashed on imdb caused an issue
that it is no longer possible to manually add content to
the videolibrary. Removing one of the checks for for a non
existing videoinfo tag is getting rid of that bug.
---
 xbmc/video/windows/GUIWindowVideoBase.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/xbmc/video/windows/GUIWindowVideoBase.cpp b/xbmc/video/windows/GUIWindowVideoBase.cpp
index 908cd2c8ff71..3a5c8b237136 100644
--- a/xbmc/video/windows/GUIWindowVideoBase.cpp
+++ b/xbmc/video/windows/GUIWindowVideoBase.cpp
@@ -398,10 +398,6 @@ bool CGUIWindowVideoBase::ShowIMDB(CFileItemPtr item, const ScraperPtr &info2, b
     CGUIDialogOK::ShowAndGetInput(CVariant{13346}, CVariant{14057});
     return false;
   }
-  
-  // If the scraper failed above and no videoinfotag was created, return
-  if (!item->HasVideoInfoTag())
-    return false;
 
   bool listNeedsUpdating = false;
   // 3. Run a loop so that if we Refresh we re-run this block
